ivadomed/config_manager.py

```diff
@@ -72,6 +75,7 @@ class ConfigurationManager(object):
     """
     def __init__(self, context_path):
         self.context_path = context_path
+        self.key_change_dict = KEY_CHANGE_DICT
```

Comment:
Is it to ensure the retrocompatibility with previous packaged models where the Modified3DUnet was called Unet3D? If so, I would recommend adding a comment to explain this at line 60.

Reply:
I added a comment in this commit: a85c18e